servers/rendering/shader_language.cpp

```diff
@@ -2975,6 +3061,7 @@ void ShaderLanguage::get_builtin_funcs(List<String> *r_keywords) {
 }
 
 ShaderLanguage::DataType ShaderLanguage::get_scalar_type(DataType p_type) {
+	// FIXME: out of sync with enum
```

Comment:
Fix and remove.

Reply:
I intended this as a warning for future editors of the code. This code is dangerous and was wrong prior to my change. I will think about how best to fix this situation.